web/src/routes/admin/api/create-user.ts

```diff
@@ -1,44 +1,34 @@
 import type { RequestHandler } from '@sveltejs/kit';
-import { serverEndpoint } from '$lib/constants';
+import { api } from '@api';
 
-export const post: RequestHandler = async ({ request, locals }) => {
-  const form = await request.formData();
+export const post: RequestHandler = async ({ request }) => {
+	const form = await request.formData();
 
-  const email = form.get('email')
-  const password = form.get('password')
-  const firstName = form.get('firstName')
-  const lastName = form.get('lastName')
+	const email = form.get('email');
+	const password = form.get('password');
+	const firstName = form.get('firstName');
+	const lastName = form.get('lastName');
 
-  const payload = {
-    email,
-    password,
-    firstName,
-    lastName,
-  }
+	const { status } = await api.userApi.createUser({
+		email: String(email),
+		password: String(password),
+		firstName: String(firstName),
+		lastName: String(lastName),
+	});
 
-  const res = await fetch(`${serverEndpoint}/user`, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-      'Authorization': `Bearer ${locals.user?.accessToken}`
-    },
-    body: JSON.stringify(payload),
-  })
-
-  if (res.status === 201) {
-    return {
-      status: 201,
-      body: {
-        success: 'Succesfully create user account'
-      }
-    }
-  } else {
-    return {
-      status: 400,
-      body: {
-        error: await res.json()
-      }
-    }
-
-  }
-}
\ No newline at end of file
+	if (status === 201) {
```

Comment:
🤔 So if I understand this right, this does a POST request to the SvelteKit server, which then does a request to the API server and then transforms the response it gets to send back to the client.

I think for API data that is requested after the client has been loaded, it should be done from the client. This can be part of other ticket.